lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -24,6 +24,12 @@ click_create_button:
   - selector:
       xpath: //button[contains(text(),'Create')]
     op: click
+click_confirm_button:
+  element:
+    selector:
+      id: confirm-action
+    type: button
+    op: click
```

Comment:
can we reuse `submit_changes` rules here?

Reply:
Yeah, we can. I also would like to change the definition of `submit_changes`